joplin/pages/official_documents_page/models.py

```diff
@@ -56,11 +57,7 @@ class OfficialDocumentPage(JanisBasePage):
         FieldPanel('slug_vi'),
         FieldPanel('date'),
         FieldPanel('authoring_office', widget=countMe),
-        FieldPanel('summary', widget=widgets.CountableWidget(attrs={
-            'data-count': 'characters',
-            'data-max-count': AUTHOR_LIMITS['document_summary'],
-            'data-count-direction': 'down'
-        })),
+        FieldPanel('summary'),
```

Comment:
Do we not care about the CountableWidget anymore? Or does that not work on RichTextFields? I just ran into this, but I forgot.

Reply:
We may still care about the CountableWidget, I should check with design about it. When I include it in the FieldPanel (the way it is currently set up in master), it reverts the panel to be a normal text field.